Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -20,4 +20,25 @@ def determine_rpath_paths
     paths += homebrew_extra_library_paths
     paths.to_path_s
   end
+
+  def x11
+    xorg = Formula["linuxbrew/xorg/xorg"]
+    if xorg.installed?
+      xorg.recursive_dependencies.each do |d|
+        dep_formula = d.to_formula
+        append_path "PKG_CONFIG_LIBDIR", dep_formula.lib/"pkgconfig"
+        append_path "PKG_CONFIG_LIBDIR", dep_formula.share/"pkgconfig"
```

Comment:
It would be interesting to find out what Shaun thinks about it but I think it might be good to **install** Xorg using `brew install xorg` but not make `xorg` a dependency of any other package and use real (sub-) packages (like `libxcb`) instead. Hence, I'm not 100% sure that the change to THIS file is needed. But I do think that the other change (to `build.rb`) is necessary.